Worktez/worktez
# Fix(task-details):Fixed reload issue in task comments

### Functionality:
Fixed reload issue in task details comments

### Solution:
Changes in task-details component.

### Risk level:

### How to test:
1.Open emulator
2.go to my dashboard
3.click on any of the task
4. add a comment
5. you can see the reload issue is fixed.
